webrtc-stats.html

```diff
@@ -2607,6 +2689,14 @@ <h2>
                   echo cancellation is applied. Calculated in decibels, as defined in [[!ECHO]]
                   (2012) section 3.14.
                 </p>
+                <p>
+                  Note: If multiple audio channels are used, metrics based on
+                  samples do not increment at a higher rate. "A sample" refers
+                  to having a sample in either channel - simultaneously having
+                  samples in multiple channels counts as a single sample. When
+                  calculating echo return loss, the channel of the least audio
+                  energy is considered for any sample.
```

Comment:
confirming if "least audio energy" was intended, we should explain what is the consequence or why we picked least. I am assuming this is not documented in any other standards.

Reply:
Correction: The greatest ERL value